homeassistant/components/vorwerk/translations/en.json

```diff
@@ -0,0 +1,22 @@
+{
```

Comment:
This file can be removed. It will be auto generated from strings.json and Lokalise